### PR TITLE
feat: apply the range of recent records to all routings & fix minor bugs

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci`, minor refactoring, etc.)
- [ ] Need discussion
- [ ] Not that difficult
- [ ] Approved feature branch merge to master

### Description
SSIA

### CODE REVIEW

File: apps/web/src/router/index.ts [hunk 1]
```diff
@@ -271,6 +272,27 @@ export class SpaceRouter {
...
             if (GTag.gtag) GTag.setPageView(to);
+            const store = SpaceRouter.router.app?.$store;
+            if (!store) return;
+            const isAdminMode = appContextStore.getters.isAdminMode;
+            const routeScope = getRouteScope(to);
+
+            if (!isAdminMode && routeScope === 'WORKSPACE') {
+                const recent = getRecentConfig(to);
+                if (recent) {
+                    console.log('recent', to);
```
Comment (yuda110) [inline apps/web/src/router/index.ts:283]: nitpick: console.log!

**Review by piggggggggy — APPROVED** [review]
Goooood~!

File: apps/web/src/router/index.ts (hunk 1 above)
Comment (sulmoJ) [reply apps/web/src/router/index.ts:283]: 와우

File: apps/web/src/common/modules/navigations/top-bar/modules/top-bar-search/modules/top-bar-search-dropdown/modules/TopBarSearchRecentListItem.vue [hunk 2]
```diff
@@ -124,9 +146,9 @@ const handleClick = () => {
                        position="bottom"
             >
                 <div class="upper-part">
-                    <span>{{ getLabelByResourceId(props.resourceId, storeState.activeTab) }}</span><span v-if="props.resourceId"
-                                                                                                         class="desc"
-                    ><span v-if="storeState.activeTab === 'dashboard'"><span class="dot">∙</span><span>{{ getDescriptionByResourceId(props.resourceId, storeState.activeTab) }}</span>
+                    <span>{{ getLabelByResourceId(state.convertResourceId, storeState.activeTab) }}</span><span v-if="state.convertResourceId"
```
Comment (yuda110) [inline apps/web/src/common/modules/navigations/top-bar/modules/top-bar-search/modules/top-bar-search-dropdown/modules/TopBarSearchRecentListItem.vue:149]: quibble: need some line break😅

**Pull request merged**